main.c

```diff
@@ -0,0 +1,138 @@
+#include <stdio.h>
+#include <string.h>
+
+enum
+{
+    data,
+    next
+};
+
+void StringListInit(char*** list)
+{
+    list[data] = NULL;
+    list[next] = NULL;
+}
+
+void StringListAdd(char*** list, char* str)
+{
+    char** node = (char**)malloc(sizeof(char*) * 2);

```

Comment:
It is better to verify input parameters if it is not NULL, then allocate memory. 
There might be dereferencing of a potential null pointer 'node'.